api/v1alpha6/conversion_test.go

```diff
@@ -44,11 +44,13 @@ func TestFuzzyConversion(t *testing.T) {
 				c.FuzzNoCustom(v1alpha7Cluster)
 
 				v1alpha7Cluster.Spec.APIServerLoadBalancer.Provider = ""
+				v1alpha7Cluster.Spec.Router = nil
 			},
 			func(v1alpha7ClusterTemplate *infrav1.OpenStackClusterTemplate, c fuzz.Continue) {
 				c.FuzzNoCustom(v1alpha7ClusterTemplate)
 
 				v1alpha7ClusterTemplate.Spec.Template.Spec.APIServerLoadBalancer.Provider = ""
+				v1alpha7ClusterTemplate.Spec.Template.Spec.Router = nil
```

Comment:
I didn't expect this to be necessary, but I can see that the test fails without it. I am investigating this.

Reply:
I've just submitted https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1517 to remove the other use of FuzzerFuncs here. Please could you look at that PR for how to do this in the converter rather than ignoring this in the test.

Please could you rebase on top of that PR? It's probably simplest to wait for it to merge first. Sorry about this!